### PR TITLE
add openweather api and render on page, more styling is needed

### PR DESCRIPTION
**Pull request by talarsen**
## Overview

This is the open weather API feature. It is currently set up so that the weather report renders the current browser's longitude and latitude. 

## Feedback Requested

1. Mostly, I am trying to get practice filling this request out and then merging it to main. 
2. The problem with the Openweather API and the fire map is that they are not customized to render for any particular fire. I am thinking that there is a way to set longitude and latitude in the database and hook them up in the components instead of hardcoding lat/long. 
3. The API is being called twice, the first time it is  400 error, and then about 5 seconds later it renders ok with 200.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ ARG NODE_ENV=production
 ARG REACT_APP_TITLE
 ARG REACT_APP_SUBTITLE
 ARG REACT_APP_INCIDENT_MAP
+ARG REACT_APP_OPENWEATHER
+ARG REACT_APP_API_URL 
+ARG REACT_APP_ICON_URL
```
Comment (ndp) [inline Dockerfile:8]: You should consider making all of these more descriptive. For example, think about how you would understand `REACT_APP_API_URL` when you come back to it in a month. Which API?

File: app/src/components/Incident.js [hunk 2]
```diff
@@ -5,15 +5,36 @@ import * as apiClient from "../apiClient";
 
 import IncidentMap from "./IncidentMap";
 import IncidentSummary from "./IncidentSummary";
+import Weather from "./Weather";
 const Incident = ({ incidentNumber }) => {
   const [incident, setIncident] = React.useState([]);
-
+  const [lat, setLat] = React.useState([]);
+  const [long, setLong] = React.useState([]);
+  const [data, setData] = React.useState([]);
```
Comment (ndp) [inline app/src/components/Incident.js:13]: I have a personal rule that `data` is never a good name for a variable. Can you use something more descriptive?

File: server/src/db.mjs [hunk 3]
```diff
@@ -11,7 +11,7 @@ export const getIncidents = () => db.any("SELECT * FROM incidents");
 //get ONE incident
 //not much different from get all but specify params in route.
 export const getOneIncident = (incident_number) =>
-  db.one("SELECT * FROM incidents WHERE incident_number = $1", [
+  db.one("SELECT * from incidents WHERE incident_number = $1", [
```
Comment (ndp) [inline server/src/db.mjs:14]: Why make this change?

**Comment by ndp** [comment]
Nice PR intro. 

I did my best to review as I would normally.

**Review by talarsen — COMMENTED** [review]
Thank you, Andy.

**Comment by ndp** [comment]
LGTM 👍

**Pull request merged**